GliaWidgets/Component/Connect/Operator/ConnectOperatorView.swift

```diff
@@ -88,6 +98,24 @@ class ConnectOperatorView: UIView {
         animationView = nil
     }
 
+    func showOnHoldView() {
+        guard let style = style.onHoldOverlay else { return }
```

Comment:
I'd like to propose to call on the beginning of this method `onHoldView?.removeFromSuperview()`. To be sure that accidentally called `showOnHoldView` will not increment amount of onHoldView subviews.